pkgs/tools/security/step-cli/default.nix

```
@@ -3,10 +3,11 @@
   buildGoModule,
   fetchFromGitHub,
 }:
-
-buildGoModule rec {
-  pname = "step-cli";
+let
   version = "0.26.1";
+in
+buildGoModule {
+  pname = "step-cli";
```

If you make the change from rec to let, you also need
```suggestion
  pname = "step-cli";
  inherit version;
```

But this doesn't seem like a net improvement IMO

---

Maybe this?
https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec

---

Okay. Either way still needs a change to build properly.